internal/guest/storage/scsi/scsi.go

```diff
@@ -358,6 +361,12 @@ func GetDevicePath(ctx context.Context, controller, lun uint8, partition uint64)
 			case <-ctx.Done():
 				return "", ctx.Err()
 			default:
+				if !logged && logTime.Before(time.Now()) {
```

Comment:
Just a thought, why not <-time.After(time.Second * 5) as a case and then just continue. Yes you would get multiple log statements but it feels like thats ok. What if it takes 15 seconds? You would see multiple logs at 5 second intervals. But thats also helpful to see how long it is taking.